src/oscar/apps/checkout/mixins.py

```diff
@@ -144,7 +139,6 @@ def place_order(self, order_number, user, basket, shipping_address,
             total=order_total,
             billing_address=billing_address,
             status=status,
-            request=request,
```

Comment:
By not passing the request the order will not be properly associated with the site it originated from.